src/Tests/PathautoTokenTest.php

```diff
@@ -42,7 +42,7 @@ public function testPathautoTokens() {
     /* @var \Drupal\pathauto\PathautoManagerInterface $manager */
     $manager = \Drupal::service('pathauto.manager');
     $manager->cleanTokenValues($replacements, $data, array());
-    $this->assertEqual($replacements['[array:join-path]'], 'test-first-arg/array-value');
+    $this->assertEqual($replacements['[array:join]'], 'test-first-arg/array-value');
```

Comment:
I think it is basically correct in the test, just the path that get's generated is wrong:

Value `test-first-arg-array-value` is equal to value `test-first-arg/array-value`.